packages/myst-frontmatter/src/utils/parseName.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { describe, expect, test } from 'vitest';
+import type { Name } from '../frontmatter/types.js';
+import { parseName, renderName, startsWithUpperCase } from './parseName.js';
+
+describe('Test name parsing', () => {
```

Comment:
would be nice to also add a comment with the reference url here
http://maverick.inria.fr/~Xavier.Decoret/resources/xdkbibtex/bibtex_summary.html#names

Reply:
Url is commented in the relevant test yml file here: https://github.com/executablebooks/mystmd/pull/586/files#diff-d593203075583406317a2f9fd53086ff0dd32c771964d351e56195307e5e9510R2